include/rmm/mr/device/cuda_async_memory_resource.hpp

```diff
@@ -29,8 +29,10 @@
 #include <limits>
 
 #if CUDART_VERSION >= 11020  // 11.2 introduced cudaMallocAsync
+#ifndef NO_RMM_CUDA_MALLOC_ASYNC_SUPPORT
```

Comment:
I think RMM macros start with `RMM`, so I think this should be `RMM_DISABLE_CUDA_MALLOC_ASYNC`

Reply:
Updated to `RMM_DISABLE_CUDA_MALLOC_ASYNC`